plugins/SimulationView/SimulationViewMenuComponent.qml

```diff
@@ -130,10 +130,53 @@ Cura.ExpandableComponent
         ComboBox
         {
             id: layerTypeCombobox
+            textRole: "text"
+            valueRole: "type_id"
             width: parent.width
             model: layerViewTypes
             visible: !UM.SimulationView.compatibilityMode
-            style: UM.Theme.styles.combobox
+
+            background: Rectangle
```

Comment:
https://github.com/Ultimaker/Cura/pull/11336/commits/4ef394fa2df94c652cb14c2886f4138750e5642a

Reply:
(Fixed in this commit, rather than the one linked above: https://github.com/Ultimaker/Cura/pull/11336/commits/2290b46bf3acc75a9346815e0d61e0d5aef94751)